RELEASES.md

```diff
@@ -10,7 +10,343 @@ of each version.
 
 v4.11.0
 ----------
-Release notes to be written.
+
+### Language features, tactics, and metaprograms
+
+* The variable inclusion mechanism has been changed. Like before, when a definition mentions a variable, Lean will add it as an argument of the definition, but now in theorem bodies, variables are not included based on usage in order to ensure that changes to the proof cannot change the statement of the overall theorem. Instead, variables are only available to the proof if they have been mentioned in the theorem header or in an **`include` command** or are instance implicit and depend only on such variables. The **`omit` command** can be used to omit included variables.
```

Comment:
```suggestion
* The variable inclusion mechanism has been changed. Like before, when a definition mentions a variable, Lean will add it as an argument of the definition, but now in theorem bodies, variables are not included based on usage in order to ensure that changes to the proof cannot change the statement of the overall theorem. Instead, variables are only available to the proof if they have been mentioned in the theorem header or in an **`include` command** or are instance implicit and depend only on such variables.
```
As far as I can tell, `omit` didn't make it into 4.11.

Reply:
It made it in with #5000

Reply:
Ah, OK - I was writing my release post draft and testing it in -rc2, which doesn't seem to have it. Just tried with -rc3, and it works. Thanks!